plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/parser/exchange/binance/v4/BinanceSortedGroupV4.java

```diff
@@ -0,0 +1,281 @@
+package io.everytrade.server.plugin.impl.everytrade.parser.exchange.binance.v4;
+
+import io.everytrade.server.model.TransactionType;
+import io.everytrade.server.plugin.impl.everytrade.parser.exception.DataIgnoredException;
+
+import java.math.BigDecimal;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import io.everytrade.server.model.Currency;
+
+public class BinanceSortedGroupV4 {
+
+    Instant time;
+
+    // beforeSum
+    Map<Currency, List<BinanceBeanV4>> rowsDeposit = new HashMap<>();
```

Comment:
if these fields should be private add `@FieldsDefault(level = PRIVATE)` on class